R/clean_names.R

```diff
@@ -53,3 +55,8 @@
   means$Parameter <- NULL
   means
 }
+
+
+.capitalize <- function(x) {
```

Comment:
[`tools::toTitleCase()`](https://stackoverflow.com/a/38527665/4198688)?

Reply:
I knew there was a function, but could not remember... `toTitleCase()` does a bit more, but for a single word, it doesn't matter and has the same result as `.capitalize()`.